docs/source/guide/configuration.rst

```diff
@@ -48,9 +50,76 @@ To set these configuration options, create a ``Config`` object with the options
 
     client = boto3.client('kinesis', config=my_config)
 
+
+Using proxies
+~~~~~~~~~~~~~
+With Boto3, you can use proxies as intermediaries between your code and AWS. Proxies can provide functions such as filtering, security, firewalls, and privacy assurance.
+
+.. _specify_proxies:
+
+Specifying proxy servers
+''''''''''''''''''''''''
+
+You can specify proxy servers to be used for connections when using specific protocols. The ``proxies`` option in the ``Config`` object is a dictionary in which each entry maps a protocol to the address and port number of the proxy server for that protocol.
+
+In the following example, a proxy list is set up to use ``proxy.amazon.com``, port 6502 as the proxy for all HTTP requests by default. HTTPS requests use port 2010 on ``proxy.amazon.org`` instead.
+
+
+.. code-block:: python
+
+    import boto3
+    from botocore.config import Config
+
+    proxy_definitions = {
+        'http': 'http://proxy.amazon.com:6502',
+        'https': 'https://proxy.amazon.org:2010'
+    }
+
+    my_config = Config(
+        'region_name': 'us-east-2',
+        'signature_version': 'v4',
+        'proxies': proxy_definitions
+    }
+
+    client = boto3.client('kinesis', config=my_config)
+
+
+.. _configure_proxies:
+
+Configuring proxies
+'''''''''''''''''''
+You can configure how Boto3 uses proxies by specifying the ``proxies_config`` option, which is a dictionary that specifies the values of several proxy options by name.  There are three keys in this dictionary: ``proxy_ca_bundle``, ``proxy_client_cert``, and ``proxy_use_forwarding_for_https``. For more information about these keys, see the `Botocore config reference <https://botocore.amazonaws.com/v1/documentation/api/latest/reference/config.html#botocore.config.Config>`_.
+
+.. code-block:: python
+
+    import boto3
+    from botocore.config import Config
+
+    proxy_definitions = {
+        'http': 'http://proxy.amazon.com:6502'
```

Comment:
This should be an https entry in order to make sense with the `proxies_config` below. TLS won't be negotiated for http calls, making the cert declaration a no-op.

Reply:
Fix pushed; I replaced the body of `proxy_definitions` with the body of the one from the previous sample, which has both HTTP and HTTPS proxies defined. Then I added a few words to the explanation, noting that the certificate will only apply to the HTTPS proxy.